### DISCUSSION TITLE
Ganache-cli issues

### DISCUSSION THREAD

**Discussion by leopashov** [smartcontractkit/full-blockchain-solidity-course-py#74] [Q&A] [answered]
Hi,
First of all thank you very much for this course!

I've run into issues using ganache-cli (I'm around the 4:20:00 mark)
ganache-cli seemed to install fine, I got the following in command line:
yarn global v1.22.11
[1/4] Resolving packages...
[2/4] Fetching packages...
[3/4] Linking dependencies...
[4/4] Building fresh packages...
success Installed "ganache-cli@6.12.2" with binaries:        
      - ganache-cli
Done in 2.59s.

But attempting to get version (>ganache-cli --version) gives me:

'ganache-cli' is not recognized as an internal or external command,
operable program or batch file.

I am using windows by the way

Thanks in advance for any help :)

**Comment by AlbertoSinigaglia** [comment, marked as answer]
Have you followed [https://www.npmjs.com/package/ganache-cli](https://www.npmjs.com/package/ganache-cli)?

**Reply by leopashov** [reply]
looked through the FAQs and found this:
https://github.com/trufflesuite/ganache/issues/978
ended up installing ganache-cli with npm (ie using 'npm install -g ganache-cli') with firewall turned off and it now seems to have worked (y)
thank you

**Reply by PatrickAlphaC** [reply]
Boom!!! Amazing work!
